algonaut_transaction/src/account.rs

```diff
@@ -93,11 +99,86 @@ impl Account {
         Ok(SignedTransaction {
             transaction: transaction.clone(),
             sig: Some(signature),
+            logicsig: None,
             multisig: None,
             transaction_id: id,
         })
     }
 
+    pub fn sign_logic_msig(
```

Comment:
I don't like these functions (+`append_to_logic_msig`) that much, for now ported from the official SDKs. See https://github.com/manuelmauro/algonaut/issues/23